datahub-web-react/src/app/settings/posts/utils.ts

```diff
@@ -16,7 +16,7 @@ export const addToListPostCache = (client, newPost, pageSize) => {
     });
 
     // Add our new post into the existing list.
-    const newPosts = [newPost, ...(currData?.listPosts?.posts || [])];
+    const newPosts = [...(currData?.listPosts?.posts || [])];
```

Comment:
How is this fixing the issue/

Reply:
This is not a network error; we're successfully creating a post. However, we're trying to add static frontend data (title and description) directly to the local cache. The issue is that the local cache requires additional fields in the new post, which causes an error and leads to the catch block being triggered. To fix this, I removed the new post and updated code to  [...(currData?.listPosts?.posts || [])] . The error occurs because the local cache needs more fields for the new post. Specifically, the new post should have these fields:{
    "urn",
    "type",
    "postType": "Post type",
    "content": {
        "contentType": "TEXT",
        "title",
        "description",
        "link": null,
        "media": null
    }
}
And currently, we're only adding the title and description to the local cache using the onCreate method as follows:
onCreate={(urn, title, description) => {
    addToListPostCache(
        client,
        {
            urn,
            properties: {
                title,
                description: description || null
            }
        },
        pageSize
    );
    setTimeout(() => refetch(), 2000);
}} 
I also added screencast how this error is occuring with the newpost
[Screencast from 02-01-24 12:08:39 PM IST.webm](https://github.com/datahub-project/datahub/assets/77378510/c8cdf93f-c1e6-4f5e-a408-4c902973b90e)